src/test/scala/com/amazon/deequ/suggestions/ConstraintSuggestionRunnerTest.scala

```diff
@@ -225,4 +241,52 @@ class ConstraintSuggestionRunnerTest extends WordSpec with Matchers with SparkCo
 
     assert(expectedConstraintSuggestionJson == actualConstraintSuggestionJson)
   }
+
+  private[this] def suggestHasDataTypeConstraintVerifyTest(data: DataFrame): Unit = {
+
+    val constraints = ConstraintSuggestionRunner()
+      .onData(data)
+      .addConstraintRules(Rules.DEFAULT)
+      .run()
+      .constraintSuggestions
+
+    val hasDataTypeConstraint = constraints
+      .toSeq
+      .head
+      ._2
```

Comment:
I believe I understand your request to "give a name to the variable" -- you want to see this value (before the `.filter`) as a named variable, correct?

Reply:
I am not sure what you mean by "use a map function". Here, `constraints` is a `Map` with only one key-value pair since there's only one column. The `.toSeq` gives us a `Seq[(Key,Value)]`: so the last `._2` is merely obtaining the value.

I could re-write as:
```
val (_, onlyColumnConstraints) = constraints.toSeq.head
val hasDataTypeConstraint = onlyColumnConstraints.filter { ... }.head
```

Is this what is desired?